web/src/components/tasks/business-formation/BusinessFormationPaginator.test.tsx

```diff
@@ -345,19 +345,20 @@ describe("<BusinessFormationPaginator />", () => {
         it("saves municipality to profile", async () => {
           const businessWithMunicipality = {
             ...business,
-            profileData: {
-              ...business.profileData,
-              municipality: generateMunicipality({ displayName: "Newark", name: "Newark" }),
+            formationData: {
+              ...business.formationData,
+              formationFormData: {
+                ...business.formationData.formationFormData,
+                addressMunicipality: generateMunicipality({ displayName: "New Town", name: "New Town" }),
```

Comment:
boulder, i don't think hard coding values is the solution

Reply:
We're not sure what you mean. The values were already hardcoded. Can you elaborate?